prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -50,13 +50,12 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: alpine:3.15.0
+      - image: bash:alpine3.14
         command:
-        - /bin/sh 
+        - /usr/local/bin/bash
```

Comment:
it might be better omitting the full system path?
Just `bash` is enough.

Reply:
That is a good point, updated to use `bash` directly.